flow-typed/npm/@sentry/react-native_v1.x.x.js

```diff
@@ -14,7 +14,70 @@
  */
 
 declare module '@sentry/react-native' {
-  declare module.exports: any;
+  // Despite being given as exact, many other options not listed here are
+  // available. See:
+  // * https://github.com/getsentry/sentry-react-native/blob/master/src/js/backend.ts
+  // * https://github.com/getsentry/sentry-javascript/blob/master/packages/browser/src/backend.ts
+  // * https://github.com/getsentry/sentry-javascript/blob/f4e59bcdd/packages/types/src/options.ts
+  //
+  // Please add them to this declaration as (and if) they're needed.
+  declare export type Options = {|
+    enabled?: boolean,
+    dsn?: string,
+    ignoreErrors?: Array<string | RegExp>,
+  |};
```

Comment:
This `flow-typed/npm/` directory is automatically maintained, with `yarn update-libdefs` (which in turn is a thin wrapper around `flow-typed update`.)

For hand-written libdefs, to keep them clear of getting clobbered the next time we run that, we put them in the parent `flow-typed/` directory. For examples that override the third-party libdefs, see `flow-typed/react-intl.js` and `git log --stat -p flow-typed/**/*document-pick*` (or replace `**/` with `{,npm/}` if you don't have `shopt -s globstar` in your bashrc.)

Reply:
Fi—

—well, this turned out to be more interesting than I thought. See new commits.